src/profile/v100/app_preferences/resource_preferences/resource_preferences.rs

```diff
@@ -0,0 +1,268 @@
+use crate::prelude::*;
+use core::hash::Hash;
+use std::hash::Hasher;
+
+/// User off-ledger preferences regarding resources.
+#[derive(
+    Debug,
+    Default,
+    Serialize,
+    Deserialize,
+    PartialEq,
+    Eq,
+    Clone,
+    derive_more::Display,
+    uniffi::Record,
+)]
+#[serde(rename_all = "camelCase")]
+#[display("{}", self.description())]
+pub struct ResourcePreferences {
+    /// A dictionary detailing the user preferences for fungible resources.
+    #[serde(default)]
+    pub fungible: HashMap<ResourceAddress, EntityFlags>,
```

Comment:
I think `fungibles`, plural, no? It is a HashMap with settings for fungibles? (Or strictly: fungible assets plural)